NPS_generation/functions.py

```diff
@@ -30,7 +30,7 @@ def clean_mol(smiles, stereochem=False, selfies=False, deepsmiles=False):
             smiles = converter.decode(deepsmiles)
         except:
             raise ValueError("invalid DeepSMILES: " + str(deepsmiles))
-    mol = Chem.MolFromSmiles(str(smiles))
+    mol = Chem.MolFromSmiles(smiles)
```

Comment:
Let's try to keep this `str` since `smiles` is a numpy array.

Reply:
Changed it.